telegram/inline/inlinekeyboardbutton.py

```diff
@@ -45,6 +45,12 @@ class InlineKeyboardButton(TelegramObject):
 
           .. versionadded:: 13.6
 
+        * Added support for mentioning users by their ID in inline keyboards. This will only work
+          in Telegram versions released after December 7, 2021. Older clients will display
+          *unsupported message*.
+
+          .. versionadded:: 13.9
+
```

Comment:
We didn't change anything here, right? Maybe rather something like "since bot API 5.5 it's allowed to …"?

Reply:
Well we usually relate to the PTB version and not the API version in the docs, not sure if I would want to break with that.

Reply:
mh, my point is more that this works just fine with older versions of our lib. The only thing that change was that Telegram started accepting more kinds of input - when they rolled out API 5.5, not when we released v13.9 :)

Reply:
hm, then maybe just the date is more fitting, then a random version number?

Reply:
Or both 🙆🏻‍♂️